research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+---
+menu: Proposals
+name: Tabs (Editor's Draft)
+path: /components/tabs
+pathToResearch: /components/tabs.research
+showInMenu: false
+---
+
+# Tabs Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+*Tabs* are a set of layered sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+
+### Background <a href="#background" id="background"></a>
+
+_Relevant historical or background information, related existing issues, etc._
+
+### Use Cases <a href="#use-cases" id="use-cases"></a>
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
+
+### Non-goals <a href="#non-goals" id="non-goals"></a>
+
+_A list of use cases, features, or functionality which are **not** goals for the component._
+
+### Features <a href="#features" id="features"></a>
+
+- **Orientation:** Allows the tab list to be oriented horizontally above the tab content or vertically to the left or right (depending on language region) of the tab content.
+- **Supplemental content:** Offers a way to add content via start and end to the left and/or right (depending on language region) of the the tab list.
```

Comment:
Strange question, but one I got asked in the past so should probably carry over: Are there any kinds of limits or definition of supplemental content?  Can it be other interactive elements?  A close or dismiss button is the most obvious example here.

Reply:
We dug quite a bit to try and find a pre-existing answer to this exact question. We have a couple of specific scenarios in FAST we needed to be able to support where there are static interactive elements to the right or left of the tab list itself. The intent with the note above is to specifically call out that this would be supported either to the right or left of the tablist - not part of it. In the case something interferes with the tabs interaction model, I think interactive content would be problematic. We decided that so long as you can still successfully navigate the tabs model per the W3C ARIA specification, there's no problem including interactive elements in those slots.

Reply:
https://github.com/WICG/open-ui/issues/190